GoogleForCreators/web-stories-wp
# Link Panel: issues with special characters

## Context



## Summary

As documented in the DOMElement class [docs](https://www.php.net/manual/en/class.domelement.php).

> The DOM extension uses UTF-8 encoding. Use utf8_encode() and utf8_decode() to work with texts in ISO-8859-1 encoding or iconv for other encodings.

## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions

1. Create a story. 
2. Create an element. 
3. Add this link - https://www.selectgame.com.br/chizuru-kagura-estara-em-the-king-of-fighters-xv-novo-trailer/
4. See that title embed with character correct. 

You could also embed other links from https://www.selectgame.com.br/ to test. 

### QA






This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #6482
